applications/openshift/networking/routes_rate_limit/rule.yml

```diff
@@ -0,0 +1,49 @@
+prodtype: ocp4
+
+title: Ensure that all Routes has rate limit enabled 
+
+description: |-
+  OpenShift has an option to set the rate limit for Routes [1] when creating new Routes.
+  All routes outside the openshift namespaces should use the
+  rate-limiting annotations.
+
+  [1] https://docs.openshift.com/container-platform/4.9/networking/routes/route-configuration.html#nw-route-specific-annotations_route-configuration
+  
+rationale: |-
+  The usage of rate limit for Routes provides basic protection against distributed denial-of-service (DDoS) attacks.
+
+# TODO: @Vincent056 need to find out which cce
+#identifiers:
+#  cce@ocp4: CCE-xxx
```

Comment:
we move them from https://github.com/ComplianceAsCode/content/blob/master/shared/references/cce-redhat-avail.txt

Since they are unique, you need to delete it from that list once you use it.